CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 Not yet released; provisionally v1.4.0 (may change).
 
+### HTTP APIs
+
+The HTTP/JSON APIs have been deprecated in favor of a pure gRPC intereface.
```

Comment:
I don't think we can call them "deprecated" when we've deleted them. I think this should just say "removed".